source/atlas-vector-search.txt

```diff
@@ -0,0 +1,133 @@
+.. _node-atlas-vector-search:
+
+================================
+Run an Atlas Vector Search Query
+================================
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: code example, semantic, nearest
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the :atlas:`Atlas Vector Search
+</atlas-vector-search/vector-search-overview/>` feature
+in the {+driver-short+}. 
+
+You can use {+vector-search+} to perform a vector search on your data stored in
+Atlas. Vector search allows you to query your data based on semantic meaning
+rather than just keyword matches, which helps you retrieve more relevant search
+results. It enables your AI-powered applications to support use cases such as
+semantic search, hybrid search, and generative search, including
+Retrieval-Augmented Generation (RAG).
+
+To learn more about {+vector-search+}, see the :atlas:`{+vector-search+}
+</atlas-vector-search/vector-search-overview/>` guide in the MongoDB Atlas
+documentation.
+
+.. important:: Feature Compatibility
+
+   To learn what versions of MongoDB Atlas support this feature, see
+   :atlas:`Limitations </atlas-vector-search/vector-search-stage/#limitations>`
+   in the MongoDB Atlas documentation.
+
+Perform a Vector Search
+-----------------------
+
+To use this feature, you must create a vector search index and index your
+vector embeddings. To learn about how to programmatically create a
+vector search index, see the :atlas:`Create an Atlas Vector Search index
+</atlas-vector-search/vector-search-type/#create-an-atlas-vector-search-index>` section in the
+Atlas documentation. To learn more about vector embeddings, see
+:atlas:`How to Create Vector Embeddings
+</atlas-vector-search/create-embeddings/>` in the Atlas documentation.
+
+After you create a vector search index on your vector embeddings, you
+can reference this index in your pipeline stage, as shown in the
+following section.
+
+Sample Data
+~~~~~~~~~~~
+
+The example on this page shows how to build an aggregation pipeline that uses the
+``$vectorSearch`` stage to perform an exact vector search on the
```

Comment:
```suggestion
``$vectorSearch`` stage to perform a vector search on the
```
The example is an ANN rather than an ENN search. Here's the documentation: https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-stage/#fields
I think it makes sense to mention the type of search at the bottom rather than here, so I'll leave a comment in the search specs!